.github/workflows/deployFreighterApiBeta.yml

```diff
@@ -0,0 +1,47 @@
+name: Deploy @stellar/freighter-api beta
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        description: Release Number
+        default: "2.0.0"
+        required: true
+      version:
+        description: Beta version
+        default: "0"
+        required: true
+jobs:
+  bump-version:
+    name: Checkout code
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: release/${{ github.event.inputs.release }}
+      - name: Build package
+        uses: actions/setup-node@v4
+        with:
+          node-version: "21"
+      - name: Version npm package
+        run: |
+          cd @stellar/freighter-api
+          yarn version --new-version ${{ github.event.inputs.release }}-beta.${{ github.event.inputs.version }}
+          cd ../../
+      - run: yarn && yarn build:freighter-api
+      - name: Publish npm package
+        run: |
+          yarn publish --access public --tag beta
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
I have an ops token to add this secret to our repo: https://github.com/stellar/ops/issues/3399